pkg/scheduler/schedule_one_test.go

```diff
@@ -2105,6 +2107,265 @@ func TestSchedulerSchedulePod(t *testing.T) {
 	}
 }
 
+func TestPvTopology(t *testing.T) {
```

Comment:
Instead, I would prefer an integration test for this.
But this is on the nit side of things.